src/components/header/headerPatientName/index.tsx

```diff
@@ -1,16 +1,20 @@
 import * as React from "react";
 import { Icon, Navbar as BPNavbar } from "@blueprintjs/core";
+import { Patient } from "types";
 interface Props {
-  patientName: string;
+  patient: Patient;
 }
 
-const HeaderPatientName = ({ patientName }: Props) => {
+const HeaderPatientName = ({ patient }: Props) => {
   return (
     <>
       <BPNavbar.Divider />
       <Icon icon="person" />
       {/* TODO : charge name dynamically */}
-      <div> {patientName} </div>
+      <div>
+        {patient.lastName && patient.lastName + " "}
```

Comment:
does it print "undefined" if you just do `'${patient.lastName} ${patient.firstName}'` ? the `+ " "` is rather unorthodox

Reply:
Works well with ` <div>{patient.lastName} {patient.firstName}/div>`